### ISSUE TITLE
BiVAECF Validation Loss

### ISSUE THREAD

**Issue by Rico2000** [PreferredAI/cornac#427]
Hey,

i want to track the loss of the validation dataset during Training. 
At the current version of BiVAECF its not possible to do this.  
Futhermore in my oppinion it would be good to track both validation ans training loss and provide the possbility to acces it after Training for all models. 

Thanks

**Comment by tqtg** [comment]
For this experimental purpose, you might need to modify training code of the model. The library doesn't support this out of the box.
